_includes/header.html

```diff
@@ -7,7 +7,7 @@
 				<img class="nav-logo dark-logo" src="/assets/logo_dark.svg" width="136" height="48" alt="Godot Engine">
 			</a>
 			<label for="nav_toggle_cb" id="nav_toggle_btn">
-				<img src="/assets/hamburger.svg" width="24" height="24" alt="">
+				<img src="/assets/icons/hamburger.svg" width="24" height="24" alt="">
```

Comment:
```suggestion
				<img src="/assets/icons/hamburger.svg" width="24" height="24" alt="Menu">
```
I guess we should add an alt text here